bindings/CMakeLists.txt

```diff
@@ -44,7 +49,7 @@ target_compile_definitions(pyslang PRIVATE VERSION_INFO=${PROJECT_VERSION})
 install(
   TARGETS pyslang
   COMPONENT pylib
-  DESTINATION .)
+  DESTINATION ${Python_SITEARCH})
```

Comment:
I'm not sure about this change. The build here on its own is not enough to produce a python package you can place into the Python lib folder, you need the other pieces defined in https://github.com/MikePopoloski/pyslang -- which depends on this installing into a very particular location. Have you actually tested this change and does it work correctly for your use case?

Reply:
> I'm not sure about this change. The build here on its own is not enough to produce a python package you can place into the Python lib folder, you need the other pieces defined in https://github.com/MikePopoloski/pyslang -- which depends on this installing into a very particular location. Have you actually tested this change and does it work correctly for your use case?



(x) Let's see a plain ```make install``` test rounds:
 * ```before``` it is putting in something like ```/usr/./pyslang.cpython-311-{...}.so``` (very bad).
 * ```after``` is ```/usr/lib64/python3.11/site-packages/usr/./pyslang.cpython-311-{...}.so``` (the legit one).
 * User can prepend a prefix {/opt, /usr/local} to this ```make install DESTDIR=/opt```

(x) The ```${Python_SITEARCH}``` comes from ```find(python develoment.Module)```.

(x) To my knowledge, the standard python lookups:
   ```*/lib64/*/site-packages``` is the right place for binary (having some ```.so```) inclusion
   ```*/lib/*/site-packages``` is the right place for pure python (noarch) inclusion

(x) Also if you will ever publish it it via ```pip/pypi``` platform you will also need this properly done.

Let me know YES/NO and will drop it, but on this one just enumerated some strong arguments.

Reply:
Furthermore,

(x) Looking at ```pyslang``` pip:
```
$ pip3 install pyslang
  Created wheel for pyslang: filename=pyslang-2.0.0-cp311-cp311-linux_x86_64.whl size=2938144 
  Successfully installed pyslang-2.0.0
$ ls -l ~/.local/lib/python3.11/site-packages/pyslang
pyslang-2.0.0.dist-info/                 pyslang.cpython-311-x86_64-linux-gnu.so  
```
* Reveals the ```.so``` part reachable  */lib/python3.11/site-packages/pyslang* path.

Now, since ```pyslang``` and ```slang``` are separated (even in different repos) it is still a good to **match legit paths**.
Not sure how ```pip``` build flow finally **just moves** ```.so``` in the proper path, if want we can drop for now this proposal.